docs/guides/run-use-case.md

```diff
@@ -225,20 +225,37 @@ For details on SuperfaceClient API, please consult [OneSDK reference](/reference
 
 :::
 
-## Run the app
+## Compile Comlink source files to AST
 
-Now run the application to perform the use case and check the results:
+Comlink profile and map files (.supr and .suma) needs to be compiled to their AST form (with .ast.json extension).
+
+Run the following command to perform a one-off compilation:
 
 ```shell
-node app.js
+superface compile
+# or, if you don't have the CLI installed
+npx @superfaceai/cli@latest compile
 ```
 
+This will generate `.ast.json` files next to the existing source files linked from `super.json` file.
+
 :::caution
 
-If you have changed the maps or profiles in the meantime, don't forget to `compile` them using the [Superface CLI](https://github.com/superfaceai/cli)!
+The `.ast.json` files must be available in the runtime for OneSDK. It is recommended to commit the `.ast.json` files to version control.
 
 :::
 
+The compilation is **necessary after every** change to local `.suma` and `.supr` files.
```

Comment:
Highlight also the noun.

```suggestion
The compilation is **necessary after every change** to local `.suma` and `.supr` files.
```

Consider also moving this to the callout above or its own.